### PR TITLE
[libcommhistory] Export subscriber identity property from models. Contributes to MER#1303

### PR DESCRIPTION
**Pull request by matthewvogt**

### CODE REVIEW

File: src/contactgroupmodel.cpp [hunk 1]
```diff
@@ -333,6 +333,7 @@ QHash<int,QByteArray> ContactGroupModel::roleNames() const
     roles[BaseRole + StartTime] = "startTime";
     roles[BaseRole + Groups] = "groups";
     roles[BaseRole + DisplayNames] = "displayNames";
+    roles[BaseRole + SubscriberIdentity] = "subscriberIdentity";
```
Comment (chriadam) [inline src/contactgroupmodel.cpp]: that could come from the EVENT_PROPERTY_SUBSCRIBER_ID define in constants.h I guess, but probably not necessary / symmetric.

File: src/eventmodel.cpp [hunk 2]
```diff
@@ -442,6 +446,12 @@ QVariant EventModel::headerData(int section,
             case FromVCardLabel:
                 name = QLatin1String("vcard_label");
                 break;
+            case ReadStatus:
+                name = QLatin1String("read_status");
```
Comment (chriadam) [inline src/eventmodel.cpp]: unrelated?

**Comment by chriadam** [comment]
LGTM

File: src/eventmodel.cpp (hunk 2 above)
Comment (matthewvogt) [inline src/eventmodel.cpp]: Yeah, it must have been omitted earlier.
